torch/distributed/rpc/api.py

```diff
@@ -136,6 +136,16 @@ def _set_proceed_shutdown_signal(sequence_id):
     ), "Termination signal sequence id {} got set twice.".format(sequence_id)
     proceed_signal.set()
 
+thread_local_var = threading.local()
```

Comment:
We don't expect users to access this var, so let's add a "_" prefix to its name.

Reply:
@mrshenli If I add a _prefix the test doesnt have access to rpc._thread_local_var, and it needs that.  Is there a preferred way around this limitation?  If not then the name needs to be thread_local_var

Reply:
accessing private vars in our own tests are fine, we just don't expect users to use it. Other private vars like `_internal_rpc_pickler` are also accessed in test. Adding a `_` prefix is more like a convention to politely tell users "please don't access this var", but nothing actually prevent them from doing so.